lib/less/parser/parser.js

```diff
@@ -325,6 +325,26 @@ var Parser = function Parser(context, imports, fileInfo) {
                     }
                 },
 
+                namedColor: function () {
+                    parserInput.save();
+                    var autoCommentAbsorb = parserInput.autoCommentAbsorb;
+                    parserInput.autoCommentAbsorb = false;
```

Comment:
I added `parserInput.autoCommentAbsorb = false` because the comments were either doubled or swallowed without it (I do not remember which one).